nodejs/rest-api/index.basic.rest.js

```diff
@@ -0,0 +1,54 @@
+
+// Make sure you have Axios (https://www.npmjs.com/package/axios) installed
```

Comment:
This file we don't need to add it i think the atomic ones under the rest.api.handler.js are the ones to use for the integration tests :)